sphinx/source/developers/ledger.rst

```diff
@@ -70,10 +68,10 @@ The following diagram describes how deltas committed by the leader are written t
         end
 
 
-Reading the ledger and verifying entries
-----------------------------------------
+Reading and Verifing Ledger
+---------------------------
 
-The ledger is stored as a series of a 4 byte transaction length field followed by a transaction (as described on the :ref:`protocol` section).
+The ledger is stored as a series of a 4 byte transaction length field followed by a transaction (as described on the :ref:`developers/kv/kv_serialisation:Serialised Format` section).
```

Comment:
Likewise, I think we should make no mention of that detail outside the relevant page.